src/navigator/main-stack.tsx

```diff
@@ -79,6 +80,7 @@ export const MainStackScreen = () => {
 
   return (
     <PortalProvider>
+      {!isAuthorised && <ScreenStatusBar />}
```

Comment:
try to use this component at `Welcome` screen without `!isAuthorised` flag, I think this should fix this issue

Reply:
Sorry, it's a bad idea because this bug occurs not only at `Welcome` screen (see screenshots from https://www.goodday.work/t/eQt37S)

Reply:
Ok, so remove `ScreenStatusBar` from the `Wallet` screen and remove `!isAuthorised` flag from here to cover all cases.